tests/foreman/ui/test_discoveredhost.py

```diff
@@ -152,7 +154,9 @@ def test_positive_custom_provision_pxe_host(
     :expectedresults: Host should be provisioned and entry from
         discovered host should be auto removed.
 
-    :BZ: 2238952, 2268544, 2258024, 2025523
```

Comment:
Same here for 2238952

Reply:
Though it is not migrated but I see a Jira created for that BZ which is 20098